<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigureWin</class>
 <widget class="QDialog" name="ConfigureWin">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>443</width>
    <height>153</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configure Lekhonee</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Chotha/resource.qrc">
    <normaloff>:/pixmaps/lekhonee.png</normaloff>:/pixmaps/lekhonee.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout">
     <item>
      <layout class="QVBoxLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Server</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Username</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Password</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout">
       <item>
        <widget class="QLineEdit" name="serverTxt"/>
       </item>
       <item>
        <widget class="QLineEdit" name="usernameTxt"/>
       </item>
       <item>
        <widget class="QLineEdit" name="passwordTxt">
         <property name="text">
          <string/>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QPushButton" name="saveBttn">
       <property name="text">
        <string>Save</string>
       </property>
       <property name="icon">
        <iconset resource="../Chotha/resource.qrc">
         <normaloff>:/icons/document-save.png</normaloff>:/icons/document-save.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelBttn">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="icon">
        <iconset resource="../Chotha/resource.qrc">
         <normaloff>:/icons/dialog-cancel.png</normaloff>:/icons/dialog-cancel.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../Chotha/resource.qrc"/>
 </resources>
 <connections/>
</ui>
